.circleci/config.yml

```diff
@@ -142,8 +142,8 @@ workflows:
   version: 2
   commit:
     jobs:
-      - lint-and-unit-test
-      - test-coverage
+      - security-analysis:
+          context: colony-network-context
```

Comment:
I don't understand this change -- shouldn't the on-commit hook run the unit and coverage tests?

Reply:
This is a [commit to be dropped before merging ](https://github.com/JoinColony/colonyNetwork/pull/623/commits/ac64c18d6d5608194cb8cb168a6a3fce9570ee7a) for the purposes of actually running `security-analysis` though CI (as it runs nightly only)

Reply:
Fair fair